aten/src/ATen/core/op_registration/infer_schema.h

```diff
@@ -94,7 +94,7 @@ FunctionSchema createFunctionSchemaFromTraits(std::string name, std::string over
 }
 
 template<class FuncType>
-FunctionSchema inferFunctionSchema(std::string name, std::string overload_name) {
+FunctionSchema inferFunctionSchema(std::string&& name, std::string&& overload_name) {
   return detail::createFunctionSchemaFromTraits<guts::infer_function_traits_t<FuncType>>(std::move(name), std::move(overload_name));
```

Comment:
Are these moves actually doing anything?  Can't we pass this through as a reference all the way to the innermost frame?

Reply:
That's why I changed it to `&&`, now it's just passed through as a reference.

Reply:
update: changing the `FunctionSchema` constructor reduces by ~100-200kb but that seems within regular build size fluctuation.

Reply:
I implemented it in https://github.com/pytorch/pytorch/pull/26860, maybe it helps